README.md

```diff
@@ -11,7 +11,6 @@ How to start the nodejs-express-govuk-axios application
 
 1. Run `npm install` to build your application
 1. Start application with `API_URL=http://localhost:8080/ && npm start`
-1. To check that your application is running enter url `http://localhost:3000/employees`
 
 Tests
```

Comment:
Line 31, should the image link be changed? (it won't let me comment on that line...)

Reply:
Good spot!